app/services/data/get-app-court-reports.js

```diff
@@ -0,0 +1,12 @@
+const knex = require('../../../knex').appSchema
+
+module.exports = function (startingId, maxId, reportId) {
+  var selectCols = [
+    'workload_owner_id AS workloadOwnerId',
+    'id'
+  ]
```

Comment:
Not sure `selectCols` is needed. Think it adds unnecessary code when all other info is straight into the query

Reply:
yip, changed